src/views/newConnection/base.tsx

```diff
@@ -15,6 +15,10 @@ export type FormFeedback =
       type: 'error'
       error: string
     }
+  | {
+      type: 'success'
+      [k: string]: string
```

Comment:
thius could be debatable

Reply:
I don't see how a record type could ever be useful here. Is this not just a `message`?

Reply:
This will replace `success` as per last comment